x/authz/autocli.go

```diff
@@ -0,0 +1,8 @@
+package authz
+
+import autocliv1 "cosmossdk.io/api/cosmos/autocli/v1"
+
+var AutoCLIOptions = &autocliv1.ModuleOptions{
+	Query: &autocliv1.ServiceCommandDescriptor{Service: _Query_serviceDesc.ServiceName},
+	Tx:    &autocliv1.ServiceCommandDescriptor{Service: _Msg_serviceDesc.ServiceName},
+}
```

Comment:
could we add a quick few sentences on what each of these do in a doc. We can expand on it later.